tests/client_fixture.py

```diff
@@ -51,6 +52,18 @@ def profile_test_client():
     Base.metadata.drop_all(bind=test_engine)
 
 
+@pytest.fixture(scope="session")
```

Comment:
Maybe we should use the general test client? :)

Reply:
I used get_db in 'home' function (because I added the quote to the home page), so I needed to have a specific test client for home to replace the get_db with the get_test_db. Also refers to the next comment- I could not use the regular test client and had to change the test code.

I don't know yet what will be appeared at the home page, but I wanted that when the user enters the main calendar page he will see the quote, so I chose to display it at 'home' for now.. Do you think I should move it?
Thank you!

Reply:
> I used get_db in 'home' function (because I added the quote to the home page), so I needed to have a specific test client for home to replace the get_db with the get_test_db. Also refers to the next comment- I could not use the regular test client and had to change the test code.

Sababa, but I see there are already 3 similar code segments :$
Can you extract most of the code to a single function and refactor all the functions to just call it?

Reply:
> I don't know yet what will be appeared at the home page, but I wanted that when the user enters the main calendar page he will see the quote, so I chose to display it at 'home' for now.. Do you think I should move it?

That's OK, the `home` page will be refactored soon